force-app/main/default/classes/ESA_cls_ExternalCommitteeTriggerHandler.cls

```diff
@@ -0,0 +1,61 @@
+/**********************************************************************************************
+* @Author:      Vasanthi D 
+* @Date:        11 Sep 2023
+* @Description: The purpose of this class is to create methods which can be used by different trigger event
+* @Revision(s): [Date] - [Change Reference] - [Changed By] - [Description]   
```

Comment:
@deepakmulamalla Please remove this if you don't have revisions

Reply:
Removed revisions from the code.